packages/cli/src/lib/packager/createDistWorkspace.ts

```diff
@@ -166,7 +176,10 @@ export async function createDistWorkspace(
         console.warn(
           `Building ${pkg.packageJson.name} separately because it is a bundled package`,
         );
-        customBuild.push({ dir: pkg.dir, name: pkg.packageJson.name });
+        const args = buildScript.includes('--config')
```

Comment:
This wouldn't work with just a simple `backstage-cli package build` right? I think in general we'll need a check with a bit more confidence. I'm thinking we could at least check if the build script starts with `backstage-cli package build`, and that `backstage.role` is `frontend`?

Reply:
The beef is a bit outside of the default context here. In line 167, handle all build scripts that are not `backstage-cli package build`, so here we know that we're `package build` and a `bundle` role. I'd not further restrict to `frontend`, we'd want to the top-level options for some backend role thing, too. Not sure how we'd get there, though :-) 

Then this check finds if `package.json` already specifies configs, and if not, passes the one from the cli.

I tested locally with different app names to see what makes it into `index.html`'s title tag.

Reply:
Ah derp, I totally read this the wrong way around 😅. Yep makes sense! 👍  Realized it also makes sense to forward the config flags even if it's a bundled backend package that we're building, as that may in turn have frontends that need config.